modules/k8s_eks_addons/templates/autoscaler_values.yaml

```diff
@@ -2,20 +2,20 @@ awsRegion: ${aws_region}
 
 autoDiscovery:
   clusterName: ${eks_cluster_id}
+  tags:
+    - k8s.io/cluster-autoscaler/enabled
+    - k8s.io/cluster-autoscaler/${infrastructurename}

```

Comment:
@tmisch, wouldn't `eks_cluster_id` be the better variable here than `infrastructurename`, compare: https://github.com/dspace-group/simphera-reference-architecture-aws/blob/1b92d1a8ab15dc982e688d9019ee6a65319e4e6f/modules/k8s_eks_addons/cluster-autoscaler.tf#L36 ?

Reply:
It should be something _human readable_, since the cluster id typically contains the _infrastructure_ name as well, this could be ok.

Reply:
Changed _infrastrucutrename_ to _eks_cluster_id_.